src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaSelection/MediaSelection.js

```diff
@@ -12,8 +12,8 @@ const ADD_ICON = 'plus';
 
 type Props = {
     locale: observable,
-    value: ?Array<string | number>,
-    onChange: (ids: Array<string | number>) => void,
+    value: ?{ids: Array<string | number>},
```

Comment:
I think we should move that into a separate type, shouldn't we? And that would go into a separate `types.js` file then.